.github/workflows/publishes.yml

```diff
@@ -10,48 +10,73 @@ on:
     - cron: '0 16 * * *' #GMT+0
 
 jobs:
-  publish:
-    name: "Publish"
+  # job 1
+  publish-to-OSSRH:
+    name: "Publish to OSSRH"
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      matrix:
-        type: [ "OSSRH", "image" ]
     steps:
       # step 1
       - name: "Checkout"
         uses: actions/checkout@v2.4.0
-
       # step 2
       - name: "Setup Java JDK"
-        uses: actions/setup-java@v2.5.0
+        uses: actions/setup-java@v3.9.0
         with:
           distribution: 'zulu'
           java-version: 8
           server-id: oss_seata
-          server-username: OSSRH_USERNAME
-          server-password: OSSRH_PASSWORD
-
-      # step 3 for type=OSSRH
-      - name: "Import GPG-SECRET-KEY, Package and Publish-To-OSSRH"
-        if: matrix.type == 'OSSRH'
+          server-username: OSSRH_USERNAME # Environment variable name for the username for authentication to the Apache Maven repository. Default is $GITHUB_ACTOR
+          server-password: OSSRH_PASSWORD # Environment variable name for password or token for authentication to the Apache Maven repository. Default is $GITHUB_TOKEN
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
```

Comment:
Why not declare it under env?

Reply:
`gpg-private-key` is the properties of the `actions/setup-java@v3.9.0`. It's the value of the `gpg-private-key`, not an environment variable name.